programs/budget/src/budget_processor.rs

```diff
@@ -36,7 +36,10 @@ fn apply_signature(
             if &payment.to == key {
                 budget_state.pending_budget = None;
                 contract_keyed_account.try_account_ref_mut()?.lamports -= payment.lamports;
-                witness_keyed_account.try_account_ref_mut()?.lamports += payment.lamports;
```

Comment:
production code replacement

Reply:
The `map_err()` can go away with, https://github.com/solana-labs/solana/pull/16750#discussion_r618788365